public/game-master/view-models/app-viewmodel.coffee

```diff
@@ -29,4 +31,6 @@ class AppViewModel
 
 		@game_session_id.value = sessionId
```

Comment:
It doesn't make any difference, but for semantic purpose I'd keep this line on the very bottom of the file since it triggers the data flow